optimum/onnxruntime/quantization.py

```diff
@@ -315,7 +315,7 @@ def quantize(
             quantization_config.nodes_to_quantize = list(nodes_to_quantize)
             quantization_config.nodes_to_exclude = list(nodes_to_exclude)
 
-        onnx_model = onnx.load(self.onnx_model_path)
+        onnx_model = onnx.load(self.onnx_model_path.as_posix())
```

Comment:
Was there not an issue with `as_posix()` lately? Not sure about this, I may be mixing things up.

Reply:
Hi Regis, after some tests, I found that onnx treats `str` better than `PosixPath` in case of large proto(with external files). It comes from the fact that [onnx._get_file_path](https://github.com/onnx/onnx/blob/e6547e8b41578c766dd2ccf298a01b2b1b6335cd/onnx/__init__.py#L50-L56) returns os.path.abspath(f.name) instead of os.path.abspath(f) which remove the folder from the path.

This means:
* When exporting onnx model with external files, the external tensors will be put outside of the folder where lies the model proto.
![image](https://user-images.githubusercontent.com/44135271/185940670-326d5309-1c70-46f5-9c0f-2e2dee8d332a.png)

* When loading a model with external files, it will also search outside of the folder.

Theoretically, we can choose either str or PosixPath, but I think that it would be more logical to have everything in the `save_dir` defined by the users
.

Actually it would be more robust with `Path(self.onnx_model_path).as_posix()`, I will make the change!